src/modality_docs.cpp

```diff
@@ -1589,8 +1589,12 @@ void ustore_docs_write(ustore_docs_write_t* c_ptr) {
                             c.error,
                             uninitialized_state_k,
                             "Keys and values is uninitialized");
-
-            tape[idx] = result;
+            try {
```

Comment:
This doesn't need a `try` block. You can check the type without it

Reply:
Fixed.